projects/fsastorefrontlib/src/core/my-account/store/effects/quote.effect.ts

```diff
@@ -109,7 +104,6 @@ export class QuoteEffects {
               return new fromActions.LoadQuoteComparisonSuccess(carts);
             }),
             catchError(error => {
-              this.showGlobalMessage('httpHandlers.cartNotFound');
```

Comment:
Won't this affect quote comparison feature as well? Was there a valid reason we had this toast message in the first place?